### ISSUE TITLE
Item Level required

### ISSUE THREAD

**Issue by LoveyGoo** [RatScanner/RatScanner#371]
i would love to see a list or something that shows u on which level u are going to need those items like lets say  for example10 bolts on level 5 for a quest or 10bolts for a hideout upgrade pretty much a more detailed ui and the flee market thing doesnt really work maybe u could check that out too (it shows not right values or not at all) yet again thanks alot for this tool it helped me tremendously cause i just started playing tarkov and everything is confusing xD

**Comment by Blightbuster** [comment]
hey thanks for submitting the PR. A more detailed UI is definitely something desirable 👍 On the topic of wrong flea prices: This might be due to the recent pre-wipe shenanigans.

**Comment by LoveyGoo** [comment]
yeah im gonna test now too to check for the flea market prices and get back to you but thanks for the quick response :)

**Comment by LoveyGoo** [comment]
still the prices arent on point on the flea market but i guess it works most of the time i just say to head if its 10k its most likely gonna be 8 or 7k which most of the times it right

**Comment by Blightbuster** [comment]
Yeah the new item scanners of tarkov.dev are still not ready yet but I'm working on that. Once they are done prices should be updated for more often 👍

**Comment by LoveyGoo** [comment]
cool cool
